setup.py

```diff
@@ -9,9 +9,9 @@
 install_requires = [
     'click==6.6',
     'botocore>=1.5.40,<2.0.0',
-    'virtualenv>=15.0.0,<16.0.0',
     'typing==3.5.3.0',
     'six>=1.10.0,<2.0.0',
+    'pip>=9'
```

Comment:
The ``--platform`` arguments to ``pip download`` he's using are new in pip 9, the original version specifier allowed versions that wouldn't work.